documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -69,13 +69,19 @@ Because the synchronization is time-based, any switchover by consumers to a pass
 
 NOTE: If you have an application written in Java, you can use the `RemoteClusterUtils.java` utility to synchronize offsets through the application. The utility fetches remote offsets for a consumer group from the `checkpoints` topic. 
 
-== Connectivity checks
+== Deciding when to use the heartbeat connector
```

Comment:
I agree with Mickael's comments -> but it seems good in general. However, as we removed the hearbeat connector from the examples -> do we also need to add an example of how to enable it to the docs?

Reply:
We show the heartbeat connector configuration in a later section, so I'm not sure we need it here.
I've intentionally kept the config out of this overview section so that it can be reused in rhel doc.
I did add this line though:

> You enable the heartbeat connector by including its configuration in the MirrorMaker 2 deployment configuration.